selfdrive/test/test_routes.py

```diff
@@ -35,473 +35,130 @@
   TOYOTA.HIGHLANDERH_TSS2,
 ]
 
-routes = {
-  "420a8e183f1aed48|2020-03-05--07-15-29": {
-    'carFingerprint': CHRYSLER.PACIFICA_2017_HYBRID,
-    'enableCamera': True,
-  },
-  "8190c7275a24557b|2020-01-29--08-33-58": {  # 2020 model year
-    'carFingerprint': CHRYSLER.PACIFICA_2019_HYBRID,
-    'enableCamera': True,
-  },
-  # This pacifica was removed because the fingerprint seemed from a Volt
-  "f1b4c567731f4a1b|2018-04-18--11-29-37": {
-    'carFingerprint': FORD.FUSION,
-    'enableCamera': False,
-  },
-  "f1b4c567731f4a1b|2018-04-30--10-15-35": {
-    'carFingerprint': FORD.FUSION,
-    'enableCamera': True,
-  },
-  "c950e28c26b5b168|2018-05-30--22-03-41": {
-    'carFingerprint': GM.VOLT,
-    'enableCamera': True,
-  },
+
+class TestRoute:
+  def __init__(self, route, car_fingerprint, *, enable_camera=True, enable_dsu=False, enable_gas_interceptor=False):
+    self.route = route
+    self.car_fingerprint = car_fingerprint
+    self.enable_camera = enable_camera
+    self.enable_dsu = enable_dsu
+    self.enable_gas_interceptor = enable_gas_interceptor
+
+
+routes = [
+  TestRoute("420a8e183f1aed48|2020-03-05--07-15-29", CHRYSLER.PACIFICA_2017_HYBRID),
```

Comment:
the other info was only for some old test we don't run anymore. we only need the route and car now, so let's revert these changes

Reply:
No problem

Reply:
Not sure why one of the routes is failing now...

Reply:
that's unrelated, it's a recent process replay regression

Reply:
Thought so. Let me know if you want me to make a PR to clean up the routes dict